include/xrpl/protocol/UintTypes.h

```diff
@@ -58,6 +58,11 @@ using Currency = base_uint<160, detail::CurrencyTag>;
 /** NodeID is a 160-bit hash representing one node. */
 using NodeID = base_uint<160, detail::NodeIDTag>;
 
+/** MPTID is a 192-bit value representing MPT Issuance ID,
+ * which is a concatenation of a 32-bit sequence (big endian)
+ * and a 160-bit account */
+using MPTID = base_uint<192>;
```

Comment:
@shawnxie999 suggests they are serialized in this order because `STAmount` is serialized in the order of currency then account ID. Whatever the reason, can you add it to this docstring, please?

Reply:
I don't think there is any relevance between `MPTID` and how `STAmount` is serialized. `MPTID` is serialized as one block of data, there is no distinction between sequence and account in serialization. It could have been account first and then sequence.

Reply:
Yeah there isn't any relevance but since we STAmount is serialized as currency + accountID, and MPT amount is serialized as as sequence + accountID. We loosely decided to serialize MPTIssuanceID in the order of sequence + accountID as well.